services/src/contexts/mod.rs

```diff
@@ -228,49 +227,38 @@ where
     }
 
     /// This method is meant to read a ml model from disk, specified by the config key `machinelearning.model_defs_path`.
-    async fn read_ml_model(
-        &self,
-        model_sub_path: std::path::PathBuf,
-    ) -> geoengine_operators::util::Result<String> {
-        let cfg = get_config_element::<crate::util::config::MachineLearning>()
-            .map_err(|_| geoengine_operators::error::Error::InvalidMachineLearningConfig)?;
+    async fn load_ml_model(&self, model_id: ModelId) -> geoengine_operators::util::Result<String> {
```

Comment:
the doc comment for this method is outdated now

Reply:
8e0eebda

---

Commit MSG:

doc: update docstring of load_ml_model